docs/topics/auth/customizing.txt

```diff
@@ -835,23 +837,18 @@ if those assumptions are met:
   default) that can be used to identify the user and a boolean field named
   ``is_active`` to prevent password resets for inactive users.
 
-Finally, the following forms are tied to
-:class:`~django.contrib.auth.models.User` and need to be rewritten or extended
-to work with a custom user model:
+If you still need customization in these forms:
```

Comment:
It seems to me that this section about customizing `UserCreationForm` and `UserChangeForm` can be removed now. Now that you've fixed the limitation on support for user classes, the example here is just a standard modification that could apply to any form.

Reply:
At https://github.com/django/django/pull/9099#discussion_r141205320, I suggested that it might be worth keeping. I'm not sure if this technique in the forms docs.

Reply:
The class 'Meta' in UserCreationForm and UserChangeForm is not mentioned in other places of the docs, so maybe this section still has value.